src/Microsoft.DocAsCode.Build.Engine/PostProcessors/PostProcessorsHandlerWithIncremental.cs

```diff
@@ -42,8 +46,150 @@ public void Handle(List<PostProcessor> postProcessors, Manifest manifest, string
                 throw new ArgumentNullException(nameof(outputFolder));
             }
 
-            // TODO: incremental things
-            _innerHandler.Handle(postProcessors, manifest, outputFolder);
+            // TODO: add logs
+            // TODO: replay warning
+            var increItems = manifest.Files.Where(i => i.IsIncremental).ToList();
+            var noneIncreItems = manifest.Files.Where(i => !i.IsIncremental).ToList();
+
+            PreHandle(manifest, outputFolder, increItems, noneIncreItems);
+            {
+                CheckNoIncrementalItems(manifest);
```

Comment:
want to double confirm no incremental items go into inner logic...

---
In reply to: [95292681](https://github.com/dotnet/docfx/pull/1136#discussion_r95292681) [](ancestors = 95292681)